# Add missing includes to fix FTBFS with GCC 4.4. See Bug#504944
Index: exempi-2.1.0/exempi/tests/utils.cpp
===================================================================
--- exempi-2.1.0.orig/exempi/tests/utils.cpp	2008-12-28 20:39:28.000000000 +0000
+++ exempi-2.1.0/exempi/tests/utils.cpp	2008-12-28 20:43:16.000000000 +0000
@@ -34,6 +34,7 @@
  * OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <cstdio>
 
 #include <boost/test/unit_test.hpp>
 
Index: exempi-2.1.0/samples/source/XMPFilesCoverage.cpp
===================================================================
--- exempi-2.1.0.orig/samples/source/XMPFilesCoverage.cpp	2008-12-28 20:39:28.000000000 +0000
+++ exempi-2.1.0/samples/source/XMPFilesCoverage.cpp	2008-12-28 20:43:16.000000000 +0000
@@ -17,6 +17,7 @@
 #include <errno.h>
 #include <time.h>
 #include <cstring>
+#include <cstdio>
 
 #define TXMP_STRING_TYPE std::string
 #define XMP_INCLUDE_XMPFILES 1
Index: exempi-2.1.0/source/common/XML_Node.cpp
===================================================================
--- exempi-2.1.0.orig/source/common/XML_Node.cpp	2008-12-28 20:39:28.000000000 +0000
+++ exempi-2.1.0/source/common/XML_Node.cpp	2008-12-28 20:43:16.000000000 +0000
@@ -11,6 +11,7 @@
 
 #include <string.h>
 #include <cstring>
+#include <cstdio>
 #include <map>
 
 // ! Can't include XMP..._Impl.hpp - used by both Core and Files.
